components/31-molecules/spotlight/spotlight.twig

```diff
@@ -11,7 +11,8 @@
       {% set options = {
         title: 'Link title',
         link: link.url,
-        text: link.text
+        text: link.text,
+        background: 'dark-background'
```

Comment:
Ik zou de parent class op een container zetten (de blauwe container) en niet rond de link zoals in dit geval. Op die manier omvat die alle content op het donkere vlak.
Maw die background class moet doorvloeien naar cta-block (lijn 23) (en ik denk zelfs feature-block uiteindelijk).